frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx

```diff
@@ -139,33 +137,59 @@ export function SupportedPlatforms(props: {
     )
 }
 
+export function Since(props: {
```

Comment:
added this alternative to the support platforms component so we can call out subtly that there's a minimum version

(in future we should actively warn folk who are using unsupported settings but...)